examples/run_bigquery_faker.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+"""
+Usage:
+    poetry install
+    poetry run python examples/run_bigquery_faker.py
+"""
+
+from __future__ import annotations
+
+import json
+import os
+import tempfile
+import warnings
+
+from google.cloud import secretmanager
+
+import airbyte as ab
+from airbyte.caches.bigquery import BigQueryCache
+
+
+warnings.filterwarnings("ignore", message="Cannot create BigQuery Storage client")
+
+
+# load secrets from GSM using the GCP_GSM_CREDENTIALS env variable
+secret_client = secretmanager.SecretManagerServiceClient.from_service_account_info(
+    json.loads(os.environ["GCP_GSM_CREDENTIALS"])
+)
+
+# to-do: make sure database & account id are read from PY_AIRBYTE_BIGQUERY_CREDS
```

Comment:
nit.. :) change the comment to match the secret name we are using

Reply:
Great catch. Thanks! I think we can actually remove this comment...

```suggestion
```